config/data_init.json

```diff
@@ -0,0 +1,11 @@
+{   
+    "data_splitter":{
+        "ingest_path":"S:/NEU/Courses_Academic/IE7374_MLOps/Ecommerce-Data-MLOps/data/data.csv",
```

Comment:
Please rework this. Could be a potential issue later on

Reply:
Yes, I will remove it.